### PR TITLE
Add map with associations

### PR DESCRIPTION
**Pull request by taniaprocha**
- Add leaflet map to page with the locations
- Filter locations cards with map location
- Add react-leaflet plugin to gatsby

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,18 +41,23 @@
...
     "gatsby-plugin-postcss": "^2.0.0",
     "gatsby-plugin-react-helmet": "^3.0.0",
+    "gatsby-plugin-react-leaflet": "^2.0.0-alpha.3",
     "gatsby-plugin-sharp": "^2.0.5",
     "gatsby-plugin-sitemap": "^2.0.1",
     "gatsby-source-contentful": "^2.1.0-alpha.e328d33f",
     "gatsby-source-filesystem": "^2.0.5",
     "gatsby-transformer-sharp": "^2.1.1",
+    "leaflet": "^1.3.4",
+    "lodash": "^4.17.11",
```
[the remark this answers is not in the document]
Comment (taniaprocha) [reply package.json]: We also can import only reduce with lodash like this:`import reduce from "lodash/reduce"`
https://www.blazemeter.com/blog/the-correct-way-to-import-lodash-libraries-a-benchmark

[the remark this answers is not in the document]
Comment (jfranciscosousa) [reply package.json]: Yes, but you can't do this `import {get, reduce, map} from "lodash-es"`. Thing is, @filipebfernandes is already using it this way.

[the remark this answers is not in the document]
Comment (taniaprocha) [reply package.json]: Yes you can `import { map, uniq, tail, times } from "lodash"`.

[the remark this answers is not in the document]
Comment (taniaprocha) [reply package.json]: They said that `The result is that the build size is much bigger with lodash-es. So, I don’t see any reason to use it.`

[the remark this answers is not in the document]
Comment (taniaprocha) [reply package.json]: And the best way to minimize the size with `lodash` is import only the used function, one by one.

[the remark this answers is not in the document]
Comment (jfranciscosousa) [reply package.json]: Ok nice, we'll have to remove `lodash-es` then later.

**Comment by taniaprocha** [comment]
[Add mapa](https://trello.com/c/yTKjsddi/3-add-mapa)

**Pull request merged**